content/overview/overview/why-use-cloudgov.md

```diff
@@ -11,22 +11,22 @@ aliases:
 
 ## 1. Security
 
-We are building on the open source [Cloud Foundry project](http://www.cloudfoundry.org/). By using Cloud Foundry’s “stemcell” capability, we can deploy all apps on a [hardened](https://github.com/18F/cg-harden-boshrelease) operating system image that is tightly audited for compliance with federal standards. We can also centrally scan and audit the platform itself, as well as applications running on it.
+cloud.gov is built on the public [Cloud Foundry project](http://www.cloudfoundry.org/). Cloud Foundry’s “stemcell” capability allows cloud.gov to deploy all apps on a [hardened](https://github.com/18F/cg-harden-boshrelease) operating system image that is tightly audited for compliance with federal standards. The cloud.gov team can centrally scan and audit the platform itself, as well as applications running on it. We are continually improving the security of the platform, which in turn centrally improves the security of your products that are running on the platform.
```

Comment:
"The cloud.gov team can centrally scan and audit the platform itself, as well as applications running on it." - right now customers are formally responsible for their own scanning and auditing, so I recommend just removing this sentence for now.